source/material_model/interface.cc

```diff
@@ -163,6 +175,16 @@ namespace aspect
     }
 
 
+
```

Comment:
Remove this empty line please, the other functions have 2 empty lines between them.